raianalysis/README.md

```diff
@@ -0,0 +1,16 @@
+# Responsible AI Analysis SDK for Python
+
+### This package has been tested with Python 3.5, 3.6 and 3.7
```

Comment:
why not 3.8? We shouldn't really bother with 3.5 at this point IMO

Reply:
It actually says 3.6-3.8 in the setup file

Reply:
done